test/bigtest/network/start.js

```diff
@@ -0,0 +1,105 @@
+/* eslint global-require: off, import/no-mutable-exports: off */
```

Comment:
@K-Felk please see my comment above. I think we should be able to get rid of this file.